docs/pages/architecture/session-recording.mdx

```diff
@@ -10,18 +10,38 @@ configuration options that apply to session recordings.
 
 The content that is captured depends on the type of the session.
 
-- SSH sessions: Teleport captures the entire pseudo-terminal (PTY) output
-  of the session.
-- Kubernetes sessions: Teleport captures the entire PTY output for
-  `kubectl exec` invocations.
-- Desktop sessions: Teleport captures the contents of the desktop screen
-  and any mouse input. Teleport does not capture keystrokes in the remote
-  desktop.
-- App sessions: Teleport captures a stream of `app.session.request`
-  audit events related to application access. The audit events are
-  bucketed into 5-minute time intervals informally referred to as "chunks."
-- Database sessions: Teleport captures a stream of audit events
-  related to the database being accessed.
+### SSH sessions
+
+Teleport captures the entire pseudo-terminal (PTY) output of the session. The
+intention is for session recording to document what a user saw when they ran a
+session.
+
+Note that the design of session recording carries some security risks if you
+have not adequately hardened your servers. Namely, users can conceal terminal
```

Comment:
```suggestion
Note that the design of session recording carries some security risks.
Namely, users can conceal terminal
```

"if you have not adequately hardened your servers" comes off as holier than thou.  The hardening neither easy nor user friendly, so lets let the later enahnced session recommendation do the lifting here.